pkg/controller/observabilityendpoint/observabilityendpoint_controller_test.go

```diff
@@ -311,8 +311,8 @@ func TestObservabilityAddonController(t *testing.T) {
 }
 
 func TestCreateOCPClient(t *testing.T) {
-	_, err := createOCPClient()
-	if err == nil {
+	c, err := createOCPClient()
+	if err == nil && c == nil {
```

Comment:
why we check `err == nil`? I think if err is nil, the func  should work normally.

Reply:
maybe we can use this `if err != nil || c == nil` to check test failed.